{
  try {
    Assert.assertSame(Boolean.TRUE,this.extractor.getValue(this.bean));
  }
 catch (  final Exception e) {
    fail("Should not throw an exception");
  }
}
